stdlib/2and3/colorsys.pyi

```diff
@@ -0,0 +1,10 @@
+# Stubs for colorsys
+
+from typing import Tuple
+
+def rgb_to_yiq(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
```

Comment:
As every function working with float should work with complex, right? And it is also undocumented, added in 04458e6.